### PR TITLE
Fix outdated `web_accessible_resources`

### PR DESCRIPTION
**Pull request by AntonOfTheWoods**
The information appears to be out of date, as it suggests the only currently available key is `resources`. However, if only `resources` is provided then up-to-date Chrome (and Edge) both give the following error:

Invalid value for 'web_accessible_resources[0]'. Entry must at least have resources, and one other valid key.
Could not load manifest.

Fixes https://github.com/GoogleChrome/developer.chrome.com/issues/820

Changes proposed in this pull request:

- Update docs

### CODE REVIEW

**Comment by AntonOfTheWoods** [comment]
@googlebot I signed it!

File: site/en/docs/extensions/mv3/intro/mv3-migration/index.md
Comment (dotproto) [inline site/en/docs/extensions/mv3/intro/mv3-migration/index.md:258]:
```suggestion
other sites or extensions can access extension resources. See the [web 
accessible resources](/docs/extensions/mv3/manifest/web_accessible_resources/)
documentation for usage information.
```

Rather than include usage information here, we should defer to the canonical documentation for this capability.

**Review by dotproto — CHANGES_REQUESTED** [review]
Thanks for creating this PR. I'd like to make a couple tweaks to what you have here, but once these are merged I think we'll be good to accept this PR.

**Comment by AntonOfTheWoods** [comment]
Not much left from me in there but LGTM and thanks!

**Review by dotproto — APPROVED** [review]

**Pull request merged**
